src/librustdoc/clean/types.rs

```diff
@@ -641,12 +640,11 @@ impl Item {
             asyncness: ty::Asyncness,
         ) -> hir::FnHeader {
             let sig = tcx.fn_sig(def_id).skip_binder();
-            let constness =
-                if tcx.is_const_fn(def_id) || is_unstable_const_fn(tcx, def_id).is_some() {
-                    hir::Constness::Const
-                } else {
-                    hir::Constness::NotConst
-                };
+            let constness = if tcx.is_const_fn_raw(def_id) {
```

Comment:
I changed this to `raw` since we want to show all `const fn` as `const fn`, no matter their const stability attributes. A `const fn` that's unstable should get the "const: unstable" marker, but it should still show as `const fn`.